LayoutTests/platform/wk2/http/tests/security/mixedContent/secure-redirect-to-secure-redirect-to-basic-auth-insecure-image-allowCrossOriginSubresourcesToAskForCredentials.https-expected.txt

```diff
@@ -1,13 +1,12 @@
-CONSOLE MESSAGE: The page at https://127.0.0.1:8443/security/mixedContent/secure-redirect-to-secure-redirect-to-basic-auth-insecure-image-allowCrossOriginSubresourcesToAskForCredentials.https.html was allowed to display insecure content from http://localhost:8080/security/mixedContent/resources/subresource/protected-image.py.
-
-localhost:8080 - didReceiveAuthenticationChallenge - ProtectionSpaceAuthenticationSchemeHTTPBasic - Responding with testUser:testPassword
+CONSOLE MESSAGE: Cannot load image https://127.0.0.1:8443/resources/redirect.py?url=https%3A//localhost%3A8443/resources/redirect.py%3Furl%3Dhttp%3A//localhost%3A8080/security/mixedContent/resources/subresource/protected-image.py due to access control checks.
 This test loads a secure image that redirects to an secure image that redirects to an insecure image guarded by basic authentication. The image should load.
 
 On success, you will see a series of "PASS" messages, followed by "TEST COMPLETE".
 
 
-PASS did load image.
+FAIL did not load image.
 PASS successfullyParsed is true
+Some tests failed.
```

Comment:
This output and above as well does not look great as this seems like a regression.
Since WK2 should be the norm, we should update the test to have PASS instead of FAIL.